source/includes/_egress.md

```diff
@@ -107,23 +105,23 @@ curl https://api.onramp.ltd/rpc/create_egress_invoice                     \
 
 ### Request JSON Fields
 
-Field             |   Type          | Description
------------------ | --------------- | -----------
-fiat_amount       | Integer         | Eur amount to be paid denominated in cents.
-fiat_currency     | String          | The constant `"EUR"`.
-payment_ack_url   | String          | Merchant callback endpoint to confirm egress transaction. It should be a complete, well formed, url.
-user_redirect_url | String          | Where to redirect the user after the egress has been confirmed. It should be a complete, well formed, url.
-timeout_in_sec    | Integer         | When to expire the link if unused.
-offer_skin        | Egress Skin     | Specify how the offer should be displayed to the user.
-billing_details   | Billing Details | User billing details. Please, notice how **not** all parameters inside this json object are required except `merchant_customer_id`.
+Field             |   Type          | Description                                                                                                                          | Required 
+----------------- | --------------- | ------------------------------------------------------------------------------------------------------------------------------------ | --------
+fiat_amount       | Integer         | Eur amount to be paid denominated in cents.                                                                                          | Yes
+fiat_currency     | String          | `"EUR"` or `"USD"`, depending what is available on the merchants account settings.                                                   | Yes 
+payment_ack_url   | String          | Merchant callback endpoint to confirm egress transaction. It should be a complete, well formed, url.                                 | Yes
+user_redirect_url | String          | Where to redirect the user after the egress has been confirmed. It should be a complete, well formed, url.                           | Yes
```

Comment:
We are probably start to use many more url on several places, like for the new egress flow, but also for example for the redirect tile.

What about changing the type `String` to `Url` (or `Callback` or similar) and later somewhere specify what we understand for a field of type "url"/"callback" maybe with some example of what would be correct and incorrect url to us?

Reply:
Changed to Url field. I don't think we need though to specify how a callback should be as I understand Url is self explanatory.